Phonebook.Frontend/src/app/pages/dashboard/components/team/team.component.ts

```diff
@@ -29,14 +29,16 @@ export class TeamComponent implements OnInit, OnDestroy {
     if (this.currentUser != null) {
       this.organigramService.getOrganigramTree()
         .subscribe(organigram => {
-          let orgUnitOfUser = this.organigramService.getNodeForUser(this.currentUser, organigram, 0)
-          if (orgUnitOfUser != null) {
-            this.teamPersons = [
-              ...orgUnitOfUser.supervisors,
-              ...orgUnitOfUser.assistents,
-              ...orgUnitOfUser.employees,
-              ...orgUnitOfUser.learners,
-            ];
+          if (this.currentUser != null) {
```

Comment:
this should never be the case, the team view should never be shown to unauthenticated users

Reply:
yes it's outdate! nevermind :D